### DISCUSSION TITLE
Mkdocs as part of another github page

### DISCUSSION THREAD

**Discussion by alefisico** [mkdocs/mkdocs#3089] [Q&A] [answered]
Hi all

I have an organization github page with jekyll, which shows general information about my group, and I want to include a page with some documentation using mkdocs on the same website. Is this possible? 
I manage to replace the previous website entirely by following the instructions of the website (https://www.mkdocs.org/user-guide/deploying-your-docs/), but I cannot do it as part of the main website.

thanks

**Comment by pawamoy** [comment, marked as answer]
You'd have to build the MkDocs site and embed it into your Jekyll one using an iframe or something equivalent. It means the built MkDocs site would be tracked by git in your organization page repository. That's how I would do it :shrug:

**Reply by alefisico** [reply]
Thanks for the reply. Do you maybe have an example I can take a look?

**Reply by pawamoy** [reply]
I don't, sorry, but it's simply a matter of putting your built MkDocs site/folder into your Jekyll sources, and in one of your Jekyll page add an iframe sourcing the MkDocs index.html page. I'm on mobile so I'll let you search the exact syntax ^^'
